packages/core/package.json

```diff
@@ -47,8 +47,10 @@
     "css-minimizer-webpack-plugin": "^1.2.0",
     "dotenv-webpack": "^2.0.0",
     "error-overlay-webpack-plugin": "^0.4.1",
+    "esbuild": "0.9.0",
```

Comment:
`yarn add @storybook/react` automatically resorted these to be in alphabetical order. 🤷